impeller/renderer/backend/vulkan/render_pass_vk.cc

```diff
@@ -91,15 +89,8 @@ SharedHandleVK<vk::RenderPass> RenderPassVK::CreateVKRenderPass(
             attachment.texture->GetTextureDescriptor().format,        //
             attachment.texture->GetTextureDescriptor().sample_count,  //
             attachment.load_action,                                   //
-            attachment.store_action,                                  //
-            TextureVK::Cast(*attachment.texture).GetLayout()          //
+            attachment.store_action                                   //
         );
-        TextureVK::Cast(*attachment.texture)
```

Comment:
Since the render pass is cached and reused, these calls to SetLayoutWithoutEncoding were only running the first time. I moved them into the constructor so they correctly run each time.

This could be the cause of other bugs, as it could leave our own book keeping out of sync with the actual image layout.